HW python.py

```diff
@@ -0,0 +1,16 @@
+#my code:
+
+with open('text.txt') as file:
+    List=file.read().split()
```

Comment:
variables name shell not start with capital letter, also "list" is reserved word, and cant be used as variable name